kernel/devices/Rtc.cpp

```diff
@@ -0,0 +1,62 @@
+#include <devices/Rtc.h>
+#include <Cpu.h>
+#include <Log.h>
+
+namespace Kernel::Devices 
+{
+
+    uint8_t readRtcRegister(uint8_t rtcRegister)
+    {
+        CPU::PortWrite8(CMOS_ADDRESS_REGISTER, rtcRegister);
+        return CPU::PortRead8(CMOS_DATA_REGISTER);
+    }
+    
+    uint64_t readRtcTime() 
```

Comment:
the osdev wiki is reading the values at least twice in order to make sure that the read is consistent, do you want me to do the same?

Reply:
A quick look at that page seems like it might a good idea, just so we get sane values. It seems like they're just reading all the registers in a loop until they get 2 full reads that are identical.  Well worth it I think!